scripts/checkout.js

```diff
@@ -0,0 +1,56 @@
+//Assignment 8
+// Console log the values inside the form, you can build a string or build an object.
+
+// Display alert onclick submit button
+// let checkoutSubmitForm = document.getElementById('button').addEventListener("click", myFunction);
+// the line above selects the submit button and has an event listener onclick to call myFunction
+function myFunction(event) { //this queues the console log. review 'event (e)' mdn doc.
+  event.preventDefault(); //prevent page reload. review mdn doc
+  //event.target, review mdn doc
+  let custmrName = document.getElementById('custmrName');// get fields, assign variables. Place these in function myFunction.
+  let custmrContact = document.getElementById('custmrContact');
+  let custmrSt = document.getElementById('custmrSt');
+  let custmrCity = document.getElementById('custmrCity');
+  let custmrState = document.getElementById('custmrState');
+  let custmrZip = document.getElementById('custmrZip');
+  let custmrNewYes = document.getElementById('custmrNewYes');
+  let custmrNewNo = document.getElementById('custmrNewNo');
+  let pickupLoc = document.getElementById('pickupLoc');
+
+  console.log('Name: ' + custmrName.value); //logs values of field.
+  console.log('Contact: ' + custmrContact.value);
+  console.log('Address: ' + custmrSt.value);
+  console.log(custmrCity.value);
```

Comment:
If you're going to group address together, I would recommend combining all values onto the same `console.log("address:" + st + ", " + city + ", " + state + " " + zip)`.

Reply:
Oh, definitely. Good call! -S